### PR TITLE
Move include directory

### PR DESCRIPTION
**Pull request by emma-miler**
As requested on discord, this these commits were taken from the Splash PR and moved into a new one.

The main reason for the move is so that `NorthstarLauncher` also has access to `rapidjson` as it's needed for parsing JSON defining custom splash screen.

### CODE REVIEW

**Comment by emma-miler** [comment]
How to test: try to compile the solution with this PR. If it succeeds, it means its working

File: NorthstarLauncher/NorthstarLauncher.vcxproj.filters [hunk 1]
```diff
@@ -34,4 +154,4 @@
       <Filter>Resource Files</Filter>
     </Image>
   </ItemGroup>
-</Project>
+</Project>
```
Comment (pg9182) [inline NorthstarLauncher/NorthstarLauncher.vcxproj.filters:157]: Trailing newline missing?

File: NorthstarDLL/NorthstarDLL.vcxproj [hunk 2]
```diff
@@ -62,13 +62,13 @@
...
       <LanguageStandard>stdcpp20</LanguageStandard>
-      <AdditionalIncludeDirectories>$(ProjectDir)include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(SolutionDir)include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
       <SubSystem>Windows</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <EnableUAC>false</EnableUAC>
-      <AdditionalDependencies>$(ProjectDir)include\MinHook.x64.lib;$(ProjectDir)include\libcurl\lib\libcurl_a.lib;dbghelp.lib;Wldap32.lib;Normaliz.lib;version.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>$(SolutionDir)include\MinHook.x64.lib;$(SolutionDir)include\libcurl\lib\libcurl_a.lib;dbghelp.lib;Wldap32.lib;Msimg32.lib;Normaliz.lib;version.lib;%(AdditionalDependencies)</AdditionalDependencies>
```
Comment (pg9182) [inline NorthstarDLL/NorthstarDLL.vcxproj:71]: `msimg32` includes don't belong in this PR.

**Review by pg9182 — CHANGES_REQUESTED** [review]

File: NorthstarDLL/NorthstarDLL.vcxproj (hunk 2 above)
Comment (emma-miler) [reply NorthstarDLL/NorthstarDLL.vcxproj:71]: Fixed

**Review by GeckoEidechse — APPROVED** [review]
Worked fine in testing. Tested:

- [x] Booting up dedi
- [x] Launching client
- [x] Joining own dedi
- [x] Joining other random servers

**Comment by GeckoEidechse** [comment]
I assume I can merge this now, @pg9182?

**Comment by pg9182** [comment]
Looks okay to me, but I haven't ran it.

**Comment by BobTheBob9** [comment]
looks fine to me, will build and run in a bit to make sure

**Comment by GeckoEidechse** [comment]
(2 people reviewed code but never put it down as review so I didn't notice :c)
